src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -324,6 +325,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             {
                 // C# preview features.
                 case MessageID.IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction:
+                case MessageID.IDS_FeatureSealedRecordToString:
```

Comment:
>case MessageID.IDS_FeatureSealedRecordToString: [](start = 16, length = 47)

I am assuming this is a semantic check. Consider adding a comment.

Reply:
Done